src/main/drivers/light_led.h

```diff
@@ -24,7 +24,11 @@
 #include "drivers/io_types.h"
 #include "common/utils.h"
 
+#ifdef USE_RX_SPI
```

Comment:
This cannot be done as it will break consistency of parameter groups - any given version of a given parameter group needs to have the same format.

Also, the version of the parameter group will have to be incremented for this change, see https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Reply:
I'll fix that, thanks!